datamodel/testdata.d/08_05_test_data_ui.sql

```diff
@@ -59,11 +59,11 @@ INSERT INTO dcsa_im_v3_0.ops_event_timestamp_definition (
 ) VALUES (
     'd330b6f5-edcb-4e9e-a09f-e98e91deba95',
     null,
-    'TS1'
+    (select td.timestamp_id  from dcsa_im_v3_0.timestamp_definition td where td.timestamp_type_name = 'RTA-Berth (<implicit>)')
```

Comment:
The OperationsEvent has a `portCallPhaseTypeCode` of `INBD` so I feel `RTA-Berth (Inbound)` would be a more correct match.